### PR TITLE
Integrated YAML to Gulp

### PR DESCRIPTION
**Pull request by Moejoe90**
I have added to index.jsx a simple message as a test case to how things can work with json and hot reloading.

### CODE REVIEW

File: gulpfile.js [hunk 1]
```diff
@@ -7,10 +7,11 @@ const sass = require("gulp-sass");
 gulp.task("clean", () => del(["dist"]));
 
 gulp.task("page", () => {
+  let jason = require("./content.json");
```
Comment (krzkaczor) [inline gulpfile.js]: maybe name it just `content`? :D

File: gulpfile.js [hunk 2]
```diff
@@ -7,10 +7,11 @@ const sass = require("gulp-sass");
...
   // notice no return here: https://github.com/rogeriopvl/gulp-ejs/issues/86
   gulp
     .src("./page/index.ejs")
-    .pipe(ejs({ name: "world" }, {}, { ext: ".html" }).on("error", gutil.log))
+    .pipe(ejs({ name: jason.greetings }, {}, { ext: ".html" }).on("error", gutil.log))
```
Comment (krzkaczor) [inline gulpfile.js]: lets just pass whole `content` object here.

Comment (krzkaczor) [inline gulpfile.js]: You would neet to rename greetings to name, right?

File: package.json [hunk 3]
```diff
@@ -18,6 +18,7 @@
     "del": "^3.0.0",
     "file-loader": "^0.11.2",
     "font-awesome": "^4.7.0",
+    "fs": "^0.0.1-security",
```
Comment (krzkaczor) [inline package.json]: whats this?

Comment (krzkaczor) [inline package.json]: fs is build in thing in nodejs

Comment (Moejoe90) [inline package.json]: For some reason gulp doesn't recognize fs as a built in library so i had to added manually, 
It is added to dev dependancies yaml-js needs it, for some reason yml files are not read directly using require i assume support

**Pull request merged**